Simulator/source/dllmain.cpp

```diff
@@ -122,6 +122,16 @@ DLLAPI Send::Error __stdcall IbSendInit(SendType type, InitFlags flags, void* ar
                 main::send = std::move(type);
             }
             break;
+        case SendType::LogitechGHubNew:
```

Comment:
I was thinking about adding it to the if statement for SendType::AnyDriver but I think it will break things for old users.